File: climb-log-flutter/lib/climasys/features/account_view/models/get_account_details_query_response.dart
class GetAccountDetailsQueryResponse {
  final String? email;
  final String? friendlyName;
  final String userName;
  final String? profilePictureUrl;
  final String? bioText;
  final String? dateCreated;

  GetAccountDetailsQueryResponse({
    required this.email,
    required this.friendlyName,
    required this.userName,
    required this.profilePictureUrl,
    required this.bioText,
    required this.dateCreated
  });

  factory GetAccountDetailsQueryResponse.fromJson(Map<String, dynamic> json) {
    return GetAccountDetailsQueryResponse(
      email: json['email'],
      friendlyName: json['friendlyName'],
      userName: json['userName'],
      profilePictureUrl: json['profilePictureUrl'],
      bioText: json['bioText'],
      dateCreated: json['dateCreated']
    );
  }
}

File: ClimbLog.Api/ClimbLog.Application/Features/Login/Queries/GetAccountDetails/GetAccountDetailsQueryResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClimbLog.Application.Features.Login.Queries.GetAccountDetails;
public class GetAccountDetailsQueryResponse
{
    public string? Email { get; set; }
    public string? FriendlyName { get; set; }
    public required string UserName { get; set; }
    public string? ProfilePictureUrl { get; set; }
    public string? BioText { get; set; }
    public string? DateCreated { get; set; }
}
